cpt/test/integration/basic_test.py

```diff
@@ -116,6 +116,23 @@ class Pkg(ConanFile):
         self.packager.add_common_builds(shared_option_name=False)
         self.assertNotIn("lib2:shared", self.packager.items[0].options)
 
+    def test_auto_managed_subdirectory(self):
```

Comment:
This test has no asserts, and should avoid the "print"s

Reply:
Maybe check that the shared packages are generated after the add_Common_builds?

Reply:
I have force pushed a check